crates/server/README.md

```diff
@@ -0,0 +1,257 @@
+# Calimero Server
```

Comment:
Let's not use `Calimero Server` as it indicates that the company exposed some central server.

Call it `Node Server` until we define a better name.

Reply:
Updated